oeps/oep-0002-bp-repo-metadata.rst

```diff
@@ -2,24 +2,25 @@
 OEP-2: Repository Metadata
 ==========================
 
-+---------------+-------------------------------------------+
-| OEP           | :doc:`OEP-2 <oep-0002-bp-repo-metadata>`  |
-+---------------+-------------------------------------------+
-| Title         | Repository Metadata                       |
-+---------------+-------------------------------------------+
-| Last-Modified | 2019-10-29                                |
-+---------------+-------------------------------------------+
-| Author        | Calen Pennington <cale@edx.org>           |
-|               | Feanil Patel <feanil@edx.org>             |
-+---------------+-------------------------------------------+
-| Arbiter       | Calen Pennington <cale@edx.org>           |
-+---------------+-------------------------------------------+
-| Status        | Accepted                                  |
-+---------------+-------------------------------------------+
-| Type          | Best Practice                             |
-+---------------+-------------------------------------------+
-| Created       | 2016-04-21                                |
-+---------------+-------------------------------------------+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-2 <oep-0002-bp-repo-metadata>`
+   * - Title
+     - Repository Metadata
+   * - Last Modified
+     - 2019-11-14
+   * - Authors
+     - Calen Pennington, Feanil Patel, Nimisha Asthagiri
```

Comment:
Are we deliberately shifting away from including email addresses?  OEP-1 just mentions them as optional, but still has it as required for the arbiter.

Reply:
Good catch.  I'll add the email address for the arbiter.

I was influenced by the fact that later OEPs didn't include email addresses.  (Side note: going through and updating the formatting of all OEPs may be a one-time worthwhile investment.)